mkdocs.yml

```diff
@@ -7,6 +7,7 @@ nav:
         - 'ILLIXR': index.md
         - 'Getting Started': getting_started.md
         - 'Building ILLIXR': building_illixr.md
+        - 'Using ILLIXR with Cameras': using_cameras.md
         - 'ILLIXR Debugging Tips': debugging_illixr.md
         - 'ILLIXR Under Virtualization': virtualization.md
         - 'ILLIXR Plugins': illixr_plugins.md
```

Comment:
Include docs to `offload_vio` somewhere in this file. That specific doc will be updated in #405. LGTM after the change.

Reply:
The offload_vio documentation is already in master and cmake branches I believe.